src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+html {
+  box-sizing: border-box;
+}
+
 body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
The 'display: flex;' property should be declared before 'flex-direction', 'justify-content', and 'align-items' to ensure these flexbox properties are applied correctly.